src/commands/Register.ts

```diff
@@ -118,15 +118,28 @@ export class Register extends CommandClient {
 
 export async function executeRegister(registerOptions?: RegisterOptions) {
   const { app, contracts, blockchain } = getServerConfig();
-  let registerConfig: RegisterConfig | undefined = undefined;
-  if (config.has('register')) {
-    registerConfig = config.get('register');
+
+  if (!config.has('register')) {
```

Comment:
register is not necessary, it could be optional, therefore we dont need to throw an error if its not available.

Reply:
fair. I only re-arranged what was there already